neuro_extras/image.py

```diff
@@ -380,6 +383,36 @@ async def _build_image(
             raise click.ClickException(f"Failed to build image: {exit_code}")
 
 
+async def _select_build_preset(client) -> Optional[str]:
+    """
+    Try to automatically select the best available preset for image build.
+    """
+    # Require minimum 2 vCPU
+    min_cpu = 2
+    # Require minimum 4 GB RAM
+    min_mem = 4096
```

Comment:
Let's parameterize this function and put it into the utils since we will definitely use it later for data cp and artifacts transfer.